io.catenax.bom_as_specified/2.0.0/SingleLevelBomAsSpecified.ttl

```diff
@@ -0,0 +1,173 @@
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "05".
+:partQuantity a bamm:Property;
+    bamm:preferredName "Part Quantity"@en;
+    bamm:description "This is the quantity how often this part is in the item."@en;
+    bamm:characteristic :PartQuantityCharacteristic.
+:partDescription a bamm:Property;
+    bamm:preferredName "Part Description"@en;
+    bamm:description "The description of the part in the primary language of the production facility of the product owner."@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "The steering wheel is nice and round".
+:partClassification a bamm:Property;
+    bamm:preferredName "Part Classification"@en;
+    bamm:description "The part classification."@en;
+    bamm:characteristic :PartClassificationCharacteristic.
+:childCatenaXId a bamm:Property;
+    bamm:preferredName "Child CatenaXId"@en;
+    bamm:description "Describes the Catena-X ID of the child part"@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "urn:uuid:218b26f4-4a0b-4a7f-b2c1-d248927718bf ".
+:lastModifiedOn a bamm:Property;
+    bamm:preferredName "Last Modified On"@en;
+    bamm:description "The time the item was modified the last time"@en;
+    bamm:characteristic bamm-c:Timestamp;
+    bamm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime.
+:createdOn a bamm:Property;
+    bamm:preferredName "Created On"@en;
+    bamm:description "The time the item was created on"@en;
+    bamm:characteristic bamm-c:Timestamp;
+    bamm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime.
+:PartQuantityCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Part Quantity Characteristic"@en;
+    bamm:description "The characteristic of the part quantity."@en;
+    bamm:dataType :PartQuantityEntity.
+:PartClassificationCharacteristic a bamm-c:Set;
+    bamm:preferredName "Part Classification Characteristic"@en;
+    bamm:description "The characteristic of the part classification."@en;
+    bamm:dataType :PartClassificationEntity.
+:PartQuantityEntity a bamm:Entity;
+    bamm:preferredName "Part Quantity Entity"@en;
+    bamm:description "The entity encapsulating the properties describing the quantity of a part."@en;
+    bamm:properties (:measurementUnit :quantityNumber).
+:PartClassificationEntity a bamm:Entity;
+    bamm:preferredName "Part Classification Entity"@en;
+    bamm:description "The entity encapsulating the properties of the part classification."@en;
+    bamm:properties (:key :value).
+:measurementUnit a bamm:Property;
+    bamm:preferredName "Measurement Unit"@en;
+    bamm:description "Unit of measurement for the quantity of serialized objects."@en;
+    bamm:characteristic bamm-c:UnitReference;
+    bamm:exampleValue "e.g. piece, kw"^^bamm:curie.
```

Comment:
remove example value as it is not a primitive type (and it is not a valid curie)